src/main/java/frc/robot/constants/ArmConstants.java

```diff
@@ -1,21 +1,21 @@
 package frc.robot.constants;
 
 public class ArmConstants {
-  // arm ids - TBD
-  public static final int motorID = 1;
+  // TODO: arm ids - TBD (fake using 5 -- 1-4 have been taken)
+  public static final int motorID = 5;
 
-  // pid values - TBD
-  public static final double kP = -1;
-  public static final double kI = -1;
-  public static final double kD = -1;
-  public static final double kTolerance = -1;
-  public static final double minMotorPower = -1;
-  public static final double maxMotorPower = -1;
+  // TODO: PID values - TBD (fake values for now)
+  public static final double kP = 1.0;
```

Comment:
I think using a P value of zero, starting out, would be safer.

Reply:
With PID = (0, 0, 0), a simulation would never converge.

Reply:
Yes, simulation is working on a different branch.

Reply:
Safety might be better served by limiting the power to a small value such as 0.05.

I've been looking at the simulation branch. I like that it is using both the physics and mech2d. I also like the close method.